torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -850,3 +721,4 @@ def _mkldnn_weight_pack_init():
         if torch.backends.mkldnn.enabled and torch.backends.mkldnn.is_available():
             _register_weight_pack_pass()
             _recover_linear()
+            _register_unary_fusion()
```

Comment:
It's a bit weird that only unary fusion is moved here but not others. The motivation of moving it here is to leverage the "un-decomposition" in freezing but why is it "unary" specific?

Reply:
For binary ops(add, sub), they are not decomposed and do not insert type conversion before and after them, another problem is that we do an in-place optimization, which may have difficulties when tracing the in-place operation.

Reply:
As we discussed offline, let's also explore if there a simple way of match bf16 patterns after decomposition, which sounds more cleaner, no need of deciding what to move before decomposition, no need to write meta functions etc.